modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordApi.kt

```diff
@@ -450,20 +491,30 @@ class MirrordApi(private val service: MirrordProjectService, private val project
      * Reads the output (json) from stdout which contain either a success + warnings, or the errors from the verify
      * command.
      */
-    private class MirrordVerifyConfigTask(cli: String, path: String, projectEnvVars: Map<String, String>?) : MirrordCliTask<String>(cli, "verify-config", listOf("--ide", path), projectEnvVars) {
+    private class MirrordVerifyConfigTask(cli: String, configPath: String, projectEnvVars: Map<String, String>?) : MirrordCliTask<String>(cli, "verify-config", listOf("--ide", configPath), projectEnvVars) {
         override fun compute(project: Project, process: Process, setText: (String) -> Unit): String {
+            val logsService = project.service<MirrordLogsService>()
+
             setText("mirrord is verifying the config options...")
+            logsService.logInfo("mirrord is verifying the config options...")
+
             process.waitFor()
             if (process.exitValue() != 0) {
                 val processStdError = process.errorStream.bufferedReader().readText()
                 MirrordLogger.logger.error(processStdError)
+                logsService.logError("Config verification failed: $processStdError")
                 throw MirrordError.fromStdErr(processStdError)
             }
 
             val bufferedReader = process.inputStream.reader().buffered()
             val stderr = process.errorStream.reader().buffered()
-            MirrordLogger.logger.debug(stderr.readText())
+            val stderrText = stderr.readText()
+            MirrordLogger.logger.debug(stderrText)
+            if (stderrText.isNotBlank()) {
+                logsService.logInfo("Config verification stderr: $stderrText")
```

Comment:
Hmm this one I would drop the `stderr`. Also, it should be an error?

```suggestion
                logsService.logError("Config verification: $stderrText")
```